contrib/dashboards/jsonnet/dashboard_details.jsonnet

```diff
@@ -127,6 +143,22 @@ dashboard.new(
     h: 20,
   }
 )
+.addPanel(
+  graphPanel.new(
+    datasource='prometheus',
+    title="Count of src / destination subnet occurences",
```

Comment:
Change `Count` to `Connections rate` --- because the expression visualizes the rate, not the count itself.

Reply:
Done